conu/backend/docker/container.py

```diff
@@ -224,6 +225,14 @@ def get_port_mappings(self, port=None):
             if p.split("/")[0] == str(port):
                 return port_mappings[p]
 
+    def get_image_name(self):
+        """
+        return name of the container image
+
+        :return: str
+        """
+        return self.get_metadata()["Config"]["Image"]
```

Comment:
What about if these metadata ["Config"]["Image"] does not exist from some reason and trace backs? Only a question.

Reply:
Good point, I'll return `None`, if there is a problem with that.

Reply:
Done.